lib/MusicBrainz/Server/Edit/Release/ChangeQuality.pm

```diff
@@ -88,5 +89,11 @@ method accept
     );
 }
 
+sub allow_auto_edit
+{
+    my $self = shift;
+    return $self->can_amend && $self->data->{new}{quality} != $QUALITY_HIGH;
```

Comment:
Done in https://github.com/metabrainz/musicbrainz-server/pull/479/commits/3b899da260a9de1e83a6f9257f4008615f813cb3.